interface/usergroup/user_admin.php

```diff
@@ -577,7 +577,7 @@ class="form-control" value="<?php echo attr($frow['id']); ?>"><?php echo text($f
 
   </tr>
     <tr>
-        <td><span class=text><?php echo xlt('Default Billing Facility'); ?>: </span></td><td><select name=billing_facility_id style="width:150px;" class="form-control">
+        <td><span class=text><?php echo xlt('Default Billing Facility'); ?>: </span></td><td><select name="billing_facility_id" style="width:150px;" class="form-control">
```

Comment:
missed eagle scout points, dang it

Reply:
Not sure I'm following.  Do I need to fix something here?

Reply:
nope, just joking about approving it prematurely :)